.test/run.sh

```diff
@@ -92,6 +92,21 @@ mariadbclient_unix() {
 		"$@"
 }
 
+checkUserExistInMariaDB() {
+	if [ -z $1 ] ; then
+		return 1
+	fi
+
+	for user in $(mariadbclient -u root -e "SELECT User FROM mysql.user;")
```

Comment:
recommend in the SQL:
` where User='$1' and Host='$2'"` and pass the host aspect of the user in the arguments. This way you don't need to perform a loop.

Nice function idea that that should have been created a while ago.

Reply:
Done 👍🏽